cmd/start.go

```diff
@@ -45,6 +48,10 @@ Run 'colima template' to set the default configurations or 'colima start --edit'
 		app := newApp()
 		conf := startCmdArgs.Config
 
+		if startCmdArgs.Flags.Foreground {
+			defer awaitForInterruption(&app)
```

Comment:
This should be called only after a successful start. i.e. `app.Start(conf)` does not return an error.

Reply:
Updated